cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -36,10 +36,21 @@ spec:
         - name: AGENT_TOLERATION_KEY
           value: {{ .Values.agent.tolerationKey }}
 {{- end }}
+
 {{- if .Values.agent.flexVolumeDirPath }}
         - name: FLEXVOLUME_DIR_PATH
           value: {{ .Values.agent.flexVolumeDirPath }}
+{{- else }}
+{{- if (semverCompare ">=1.9-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
can you confirm what versions you were able to test locally with this fix?

Reply:
Working for me on v1.9.2+coreos.0